utils/audit/Makefile

```diff
@@ -22,7 +22,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/audit/Default
   SECTION:=utils
-  TITLE:=Audit Daemon
+  TITLE:=SELinux Audit Daemon
```

Comment:
I know this was requested in the review, but it does not make sense. The audit daemon is not directly related to SELinux. It is not a component provided by the SELinux project, and there are other things that use the Linux auditing infrastructure. So I don't think "SELinux Audit Daemon" is correct. See https://github.com/linux-audit/audit-documentation/wiki and look: there's no mention of SELinux.

Reply:
Fair enough, please rename it back. I just briefly looked at the [userspace README](https://github.com/linux-audit/audit-userspace) and it mentions SELinux, however it also states that the README is very old.

Reply:
Okay, I reverted that commit. Thank you for catching that, @tpetazzoni.